server/_install.sh

```diff
@@ -22,15 +22,15 @@ else
 fi
 
 cat > start.sh <<EOF
-#!/usr/bin/env bash
+#!/bin/bash
```

Comment:
This is the one I wanted changing, but it should be pointing to **sh** not bash, env or not shouldn't matter necessarily.